libr/core/cconfig.c

```diff
@@ -3286,6 +3303,7 @@ R_API int r_core_config_init(RCore *core) {
 	/* cfg */
 	SETBPREF ("cfg.r2wars", "false", "Enable some tweaks for the r2wars game");
 	SETBPREF ("cfg.plugins", "true", "Load plugins at startup");
+	SETCB ("cfg.analyzewrites", "false", &cb_cfganalyzewrites, "Re-analyze functions on writes");
```

Comment:
I would put it under `anal.something` as it is strictly analysis related.

Reply:
`anal.update.on.change` is a terrible name.